.github/workflows/mobilecoin-workflow-dev-setup-environment.yaml

```diff
@@ -274,6 +280,229 @@ jobs:
         rancher_url: ${{ secrets.DEV_RANCHER_URL }}
         rancher_token: ${{ secrets.DEV_RANCHER_TOKEN }}
 
+    - name: Deploy PostgreSQL instance
```

Comment:
A lot of this config was done in the generate_dev_values script or done through dependency charts and helm templating.  

Pulling this out to the actions layer does make the workflow longer, but hopefully make it easier to follow and maintain the required secret and config objects.

Reply:
Longer term I'd like to look for ways to make this less shell script-y.

Reply:
There's also quite a bit of clean up we can do for various upgrade workarounds once we get 5.1 all the way out to mainnet.